app/src/NativeBase/Components/Forms/DatePicker.tsx

```diff
@@ -0,0 +1,195 @@
+    setSelectedStartDate(value)
+    setPickerActive(false)
+    if (dayjs(value).format('DD/MM/YY') === dayjs().format('DD/MM/YY')) {
+      setTodaySelected(true)
+    } else {
+      setTodaySelected(false)
+    }
+  }
+
+  const customDatesStylesCallback = (date: Date) => {
+    if (dayjs(date).format('DD/MM/YY') === dayjs().format('DD/MM/YY')) {
+      if (colorMode === 'light') {
+        return {
+          style: {
+            backgroundColor: 'bg.100',
+            borderWidth: 1,
+          },
+        }
+      } else {
+        return {
+          style: {
+            borderColor: 'white',
+            backgroundColor: 'bgDarkMode.100',
+            borderWidth: 1,
+          },
+        }
+      }
+    }
+  }
+
+  return (
+    <VStack>
+      <Box marginTop="5" marginBottom="10" borderColor="border.100" shadow="2">
+        <VStack>
+          <Box
+            marginTop="1"
+            marginX="0.5"
+            shadow="1"
+            bg={
+              !pickerActive && colorMode === 'light'
+                ? 'lightGrey.100'
+                : 'primary.80'
+            }
+          >
+            <Text
+              paddingTop="7"
+              fontSize="xs"
+              lineHeight="7"
+              color={
+                !pickerActive && colorMode === 'light'
+                  ? 'text.100'
+                  : 'textDarkMode.100'
+              }
+              paddingLeft="6"
+            >
+              {description}
+            </Text>
+          </Box>
+          <Text
+            paddingTop="8"
+            fontSize="sm"
+            fontWeight="600"
+            lineHeight="7"
+            paddingLeft="6"
+            color={
+              !pickerActive && colorMode === 'light'
+                ? 'text.100'
+                : 'primary.100'
+            }
+          >
+            Enter Date
+          </Text>
+          <Pressable onPress={() => setPickerActive(!pickerActive)}>
+            <Input
+              marginX="6"
+              value={dayjs(value).format('DD/MM/YY')}
+              accessibilityLabel="picker for available from date"
+              borderColor="inputBorder.100"
+              isReadOnly={true}
+              size="sm"
+              borderWidth={pickerActive ? '0' : '1'}
+              borderBottomWidth="1"
+              borderBottomColor={
+                pickerActive ? 'primary.100' : 'inputBorder.100'
+              }
+              _light={{ bg: 'lighterGrey.100' }}
+              _dark={{ bg: 'bgDarkMode.100' }}
+            />
+          </Pressable>
+        </VStack>
+
+        {pickerActive ? (
+          <Box paddingTop="2">
+            <CalendarPicker
+              startFromMonday={true}
+              minDate={minimumDate}
+              maxDate={maximumDate}
+              weekdays={['M', 'T', 'W', 'T', 'F', 'S', 'S']}
+              restrictMonthNavigation={true}
+              onDateChange={handleChange}
+              date={value}
+              todayTextStyle={
```

Comment:
Small thing, but as the `fontFamily` is the same in both cases, this could be written more simply as:
```
todayTextStyle={{
                fontFamily: StaTheme.fontConfig.Poppins['500'].normal,
                color:
                  colorMode === 'light' && !todaySelected
                    ? StaTheme.colors.darkGrey['100']
                    : 'white',
              }}
```